<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WS0122985" name="calling CreateArcadeUser" tests="1" failures="1" timestamp="19 Nov 2014 20:34:49 GMT" time="270.537" errors="0">
  <testcase name="do_CreateArcadeUser on [CreateArcadeUser passed=0 failed=0 skipped=0]" time="270.467" classname="test.CreateArcadeUser">
    <failure type="java.lang.AssertionError" message="Test case CreateArcadeUser has failed due to one of its steps not returning pass status expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test case CreateArcadeUser has failed due to one of its steps not returning pass status expected [true] but found [false]
	at com.hsn.util.KeywordExecutor.ExecuteKeywords(KeywordExecutor.java:152)
	at test.CreateArcadeUser.do_CreateArcadeUser(CreateArcadeUser.java:38)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- do_CreateArcadeUser on [CreateArcadeUser passed=0 failed=0 skipped=0] -->
</testsuite> <!-- calling CreateArcadeUser -->
